app/Http/Controllers/Contacts/NotesController.php

```diff
@@ -15,13 +17,17 @@ class NotesController extends Controller
      */
     public function get(Contact $contact)
     {
+        $client = CouchDbHelper::getAccountDatabase($contact->account_id);
+
         $notesCollection = collect([]);
-        $notes = $contact->notes()->latest()->get();
+        $response = $client->limit(7)->startkey([$contact->id, []])->endkey([$contact->id])->descending(true)->include_docs(true)->getView('notes', 'byContact');
+        $notes = $response->rows;
 
-        foreach ($notes as $note) {
+        foreach ($notes as $noteArray) {
+            $note = new CouchNote((array) $noteArray->doc);
             $data = [
-                'id' => $note->id,
-                'parsed_body' => $note->parsedbody,
+                '_id' => $note->_id,
+                'parsed_body' => $note->getParsedBodyAttribute(),
```

Comment:
Laravel uses this weird stuff for accessors. That means you can access it with `$note->parsed_body`, no need to call the method directly.

Reply:
That's what I thought but as the note is not a laravel model, but is inherited from `Jenssegers\Model\Model`, it does not work.